src/vcpkg/commands.ci.cpp

```diff
@@ -210,22 +202,12 @@ namespace vcpkg::Commands::CI
             auto p = &action;
             ret->abi_map.emplace(action.spec, action.abi_info.value_or_exit(VCPKG_LINE_INFO).package_abi);
             ret->features.emplace(action.spec, action.feature_list);
-
             if (is_excluded(p->spec))
             {
                 ret->action_state_string.emplace_back("skip");
                 ret->known.emplace(p->spec, BuildResult::EXCLUDED);
                 will_fail.emplace(p->spec);
             }
-            else if (!supported_for_triplet(var_provider, p))
-            {
-                // This treats unsupported ports as if they are excluded
-                // which means the ports dependent on it will be cascaded due to missing dependencies
-                // Should this be changed so instead it is a failure to depend on a unsupported port?
-                ret->action_state_string.emplace_back("n/a");
-                ret->known.emplace(p->spec, BuildResult::EXCLUDED);
-                will_fail.emplace(p->spec);
-            }
```

Comment:
This was never hit since unsupported ports are excluded from the action plan beforehand:
https://github.com/microsoft/vcpkg-tool/blob/6040912b8dc4c0725b20510cfb2c5c79f35da168/src/vcpkg/commands.ci.cpp#L180-L184